modules/stanford_courses_importer/config/install/config_pages.type.stanford_courses_importer.yml

```diff
@@ -0,0 +1,27 @@
+langcode: en
+status: true
+dependencies:
+  module:
+    - config_pages_overrides
+third_party_settings:
+  config_pages_overrides:
+    455b050b-d304-4c74-acf4-f04767664a6b:
```

Comment:
A suggestion I have is to set the `status` of the migration config to `false` and override that value. That way if there are no url's then the migration is disabled.

Reply:
Yep, that's what i was thinking.